alarms_agent.cpp

```diff
@@ -78,6 +78,7 @@ int main (int argc, char **argv)
 
   // Log SNMP library output to syslog
```

Comment:
Should we be logging to syslog, or should we be creating /var/log/clearwater-alarms/ and logging there?